### DISCUSSION TITLE
Welcome to CaracalMob Discussions!

### DISCUSSION THREAD

**Discussion by Ivan-Khar** [AquUpd/CaracalMob#2] [Announcements]
## 👋 Welcome!
  That's a discussion where I gonna ask you:
  * New ideas
  * Bug fixes
  * Something more?

**Comment by DiHubKi** [comment]
ф л о п а

**Comment by jakobkmar** [comment]
Caracals really are the best wild cats on the planet.
And man, I love floppa.
Do you plan to continue the development on the 1.17 branch so that I can enjoy caracals in the most recent Minecraft version?

**Reply by r0yzer** [reply]
This is a very good suggestion as i both like the 1.17 and caracals. Man i love floppa.

**Reply by jakobkmar** [reply]
yeah they are so sweet
![image](https://user-images.githubusercontent.com/52456572/124366319-31be1500-dc4f-11eb-9b77-85b3ac294afd.png)

**Reply by Ivan-Khar** [reply]
Yes, but I have problems with converting 1.16 model to 1.17

**Reply by jakobkmar** [reply]
Problems of which kind?

**Reply by Ivan-Khar** [reply]
I'm not really understand structure of model class in 1.17
